### PR TITLE
Implementing i18n

### PR DESCRIPTION
**Pull request by QaysAyad**

### CODE REVIEW

File: src/componenets/Spinner/__test__/Spinner.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import Spinner from './../Spinner';
+
+test('renders spinner', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<Spinner />, div);
+});
```
Comment (MohammedAlabd) [inline src/componenets/Spinner/__test__/Spinner.test.js:9]: what did this test do?

File: src/componenets/Spinner/package.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "main": "Spinner.js"
+}
```
Comment (MohammedAlabd) [inline src/componenets/Spinner/package.json:3]: Is this file important?

Comment (QaysAyad) [reply src/componenets/Spinner/package.json]: Deleted it

File: src/componenets/Spinner/__test__/Spinner.test.js (hunk 1 above)
Comment (QaysAyad) [reply src/componenets/Spinner/__test__/Spinner.test.js]: Changed it into a snapshot test

**Pull request merged**